content/docs/lists-and-keys.md

```diff
@@ -202,13 +203,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
 
-A good rule of thumb is that elements inside the `map()` call need keys.
+Լավ կանոնն այն է, որ `map()`-ի կանչի մեջ էլեմենտները բանալիների կարիք ունեն։
```

Comment:
Իմ կարծիքով կարիք չկա `A good rule of thumb...` արտահայտությունը թարգմանելու։ Եթե նույնիս ցանկանաք, պիտի ուրիշ ձև փորձենք. ընթացիկը սխալ է։

Reply:
Տարբերակներ կա՞ն։

Reply:
![screenshot from 2019-03-06 15-18-11](https://user-images.githubusercontent.com/28267003/53879569-50cd1d80-4028-11e9-9ae1-eff59fb3a803.png)
![screenshot from 2019-03-06 15-53-02](https://user-images.githubusercontent.com/28267003/53879570-50cd1d80-4028-11e9-9ff5-db14d63e7432.png)

Reply:
Ես կառաջարկեմ "Հիմնական սկզբունքն այն է"